SwiftParcel.Services.Pricing/src/SwiftParcel.Services.Pricing.Api/Queries/GetOrderPricing.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Convey.CQRS.Queries;
+using SwiftParcel.Services.Pricing.Api.dto;
+
+namespace SwiftParcel.Services.Pricing.Api.Queries
+{
+    public abstract class GetOrderPricing : IQuery<OrderPricingDto>
+    {
```

Comment:
It doesn't work for me because of the abstract modifier

Reply:
I do not so sure it is not working in your case, but thanks, it there is no need to have it abstract.

Reply:
I don't think that OrderPrice here is necessary

Reply:
The OrderPrice here is not necessary.